clients/tests/unit/render/Fields/Mocks/MockFileFieldRenderer.js

```diff
@@ -0,0 +1,65 @@
+import {FileInput} from "../../../../../render/components/Fields/FileInput";
+import React from 'react';
+export class MockFileFieldRenderer extends React.Component {
```

Comment:
@New0 This is a fake renderer I am mounting the file input in so we can have state one level above it without dealing with the portal render.